bluebottle/bb_accounts/serializers.py

```diff
@@ -103,12 +102,23 @@ class UserProfileSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = BB_USER_MODEL
-        fields = ('id', 'url', 'username', 'first_name', 'last_name',
-                  'full_name', 'short_name', 'picture',
-                  'primary_language', 'about_me', 'date_joined', 'location',
-                  'email', 'address', 'birthdate', 'gender', 'newsletter',
-                  'campaign_notifications', 'project_count', 'donation_count',
-                  'fundraiser_count', 'task_count', 'time_spent')
+        fields = ('id', 'url', 'full_name', 'short_name', 'picture',
+                  'primary_language', 'about_me', 'location',
+                  'project_count', 'donation_count', 'date_joined',
+                  'fundraiser_count', 'task_count', 'time_spent',
+                  'website', 'twitter', 'facebook', 'skypename', )
+
+
+class UserPrivateProfileSerializer(UserProfileSerializer):
```

Comment:
Just the name yeah
On 28 May 2015 17:14, "eodolphi" <notifications@github.com> wrote:

> In bluebottle/bb_accounts/serializers.py
> <https://github.com/onepercentclub/bluebottle/pull/1604#discussion_r31243650>
> :
>
> > @@ -103,12 +102,23 @@ class UserProfileSerializer(serializers.ModelSerializer):
> >
> >      class Meta:
> >          model = BB_USER_MODEL
> > -        fields = ('id', 'url', 'username', 'first_name', 'last_name',
> > -                  'full_name', 'short_name', 'picture',
> > -                  'primary_language', 'about_me', 'date_joined', 'location',
> > -                  'email', 'address', 'birthdate', 'gender', 'newsletter',
> > -                  'campaign_notifications', 'project_count', 'donation_count',
> > -                  'fundraiser_count', 'task_count', 'time_spent')
> > +        fields = ('id', 'url', 'full_name', 'short_name', 'picture',
> > +                  'primary_language', 'about_me', 'location',
> > +                  'project_count', 'donation_count', 'date_joined',
> > +                  'fundraiser_count', 'task_count', 'time_spent',
> > +                  'website', 'twitter', 'facebook', 'skypename', )
> > +
> > +
> > +class UserPrivateProfileSerializer(UserProfileSerializer):
>
> Do you mean naming wise? Or do the ManageSerializers do other special
> things?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/onepercentclub/bluebottle/pull/1604/files#r31243650>.
>